# https://gmplib.org/list-archives/gmp-devel/2025-August/006311.html
#
diff --git a/gmp-h.in b/gmp-h.in
index 8598e97..6649434 100644
--- a/gmp-h.in
+++ b/gmp-h.in
@@ -101,8 +101,13 @@ see https://www.gnu.org/licenses/.  */
    make use of that.  Probably more trouble than it's worth.  */

 #if defined (__GNUC__)
+#ifndef __MINGW32__
 #define __GMP_DECLSPEC_EXPORT  __declspec(__dllexport__)
 #define __GMP_DECLSPEC_IMPORT  __declspec(__dllimport__)
+#else
+#define __GMP_DECLSPEC_EXPORT
+#define __GMP_DECLSPEC_IMPORT
+#endif
 #endif
 #if defined (_MSC_VER) || defined (__BORLANDC__)
 #define __GMP_DECLSPEC_EXPORT  __declspec(dllexport)
